<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPaymentMethod</class>
 <widget class="QDialog" name="AddPaymentMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Método de Pago</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo general del fondo */
QWidget {
    background-color: #F5F5F5; /* Fondo claro */
    color: #333333; /* Texto en gris oscuro */
    font-family: Arial, sans-serif;
    font-size: 14px;
}

/* Estilo de los botones */
QPushButton {
    background-color: #4CAF50; /* Verde oscuro */
    color: #FFFFFF; /* Texto en blanco */
    border: none;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    font-size: 14px;
    margin: 4px 2px;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #45a049; /* Verde más claro al pasar el ratón */
}

QPushButton:pressed {
    background-color: #388E3C; /* Verde más oscuro al presionar */
}

/* Estilo de los labels */
QLabel {
    color: #333333; /* Texto en gris oscuro */
    font-size: 14px;
}

/* Estilo de los campos de entrada */
QLineEdit, QDateEdit, QComboBox {
    padding: 5px;
    border: 1px solid #4CAF50; /* Borde verde oscuro */
    border-radius: 4px;
    background-color: #FFFFFF; /* Fondo blanco */
    color: #333333; /* Texto en gris oscuro */
}

QLineEdit:focus, QDateEdit:focus, QComboBox:focus {
    border: 1px solid #45a049; /* Borde verde más claro al enfocar */
}

/* Estilo de las tablas */
QTableWidget, QListWidget {
    border: 1px solid #4CAF50; /* Borde verde oscuro */
    border-radius: 4px;
    background-color: #FFFFFF; /* Fondo blanco */
    color: #333333; /* Texto en gris oscuro */
    padding: 5px;
}

QHeaderView::section {
    background-color: #4CAF50; /* Fondo verde oscuro */
    color: #FFFFFF; /* Texto en blanco */
    padding: 4px;
    border: 1px solid #3E3E3E; /* Borde oscuro */
}

/* Estilo de los elementos seleccionados en las tablas */
QTableWidget::item:selected, QListWidget::item:selected {
    background-color: #45a049; /* Fondo verde más claro */
    color: #FFFFFF; /* Texto en blanco */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutNombre">
     <item>
      <widget class="QLabel" name="labelNombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputNombre">
       <property name="placeholderText">
        <string>Nombre del Método de Pago</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTipo">
     <item>
      <widget class="QLabel" name="labelTipo">
       <property name="text">
        <string>Tipo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboTipo">
       <item>
        <property name="text">
         <string>Cuenta Bancaria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tarjeta de Débito</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tarjeta de Crédito</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutNumero">
     <item>
      <widget class="QLabel" name="labelNumero">
       <property name="text">
        <string>Número:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputNumero">
       <property name="placeholderText">
        <string>Número del Método de Pago</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFechaVencimiento">
     <item>
      <widget class="QLabel" name="labelFechaVencimiento">
       <property name="text">
        <string>Fecha de Vencimiento:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="inputFechaVencimiento">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
